uwblueprint/focus-on-nature
# feat: update registration API with new camper model changes

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Tasks-38b49f83f6ce4b07ae188dabb37631e8?p=4885ea4ec8f04377ab6268b2d3a0ee6d)



## Implementation description
* Updated the `camperValidator` in correspondence to new updates to the camper model and DTO
* Updated the camper service - camper creation, get campers 



## Steps to test
1. Test creating a camper with the new camper DTO structure, ensure that camper is created in the database with no errors. Also, ensure that `formResponses` is formatted correctly



## What should reviewers focus on?
* No type errors that arise from the new model changes.
* I modified the interface and schema definition for the camper for just the `formResponses` as I was getting type errors, but now it is fully functional. Simply had to define `formResponses` in the interface as an object instead of Mixed type


## Checklist
